data/battle_scripts_1.s

```diff
@@ -5129,11 +5129,19 @@ BattleScript_LocalBattleLost::
 	jumpifbattletype BATTLE_TYPE_EREADER_TRAINER, BattleScript_LocalBattleLostEnd
 	jumpifhalfword CMP_EQUAL, gTrainerBattleOpponent_A, TRAINER_SECRET_BASE, BattleScript_LocalBattleLostEnd
 BattleScript_LocalBattleLostPrintWhiteOut::
+    jumpifbattletype BATTLE_TYPE_TRAINER, BattleScript_LocalBattleLostEnd
 	printstring STRINGID_PLAYERWHITEOUT
-	waitmessage B_WAIT_TIME_LONG
+	waitmessage 0x40
+	getmoneyreward
 	printstring STRINGID_PLAYERWHITEOUT2
-	waitmessage B_WAIT_TIME_LONG
+	waitmessage 0x40
+	end2
 BattleScript_LocalBattleLostEnd::
+    printstring STRINGID_PLAYERFAINTLOSTMONEY
+	waitmessage 0x40
+	getmoneyreward
+	printstring STRINGID_PLAYERPAIDPRIZEMONEY
+	waitmessage 0x40  
```

Comment:
It seems that the constant 0x40 now has a name (B_WAIT_TIME_LONG). Please use it.